scss/mixins/_hover.scss

```diff
@@ -8,16 +8,15 @@
 //    }
 //  }
 //  @else {
-    &:hover { @content }
```

Comment:
This one doesn't need to change. Or we just remove all the commented out stuff here; it's been brought up a few times now.